README.md

```diff
@@ -1,85 +1,135 @@
 # tar_scm (OBS source service) [![Build Status](https://travis-ci.org/openSUSE/obs-service-tar_scm.png?branch=master)](https://travis-ci.org/openSUSE/obs-service-tar_scm)
 
-This is an [Open Build Service](http://openbuildservice.org/) source
-service. It uses an SCM client to checkout or update a package by
-creating a source tarball from a source code repository.
+This is the git repository for
+[openSUSE:Tools/obs-service-tar_scm](https://build.opensuse.org/package/show/openSUSE:Tools/obs-service-tar_scm),
+which provides several [source
+services](http://openbuildservice.org/help/manuals/obs-user-guide/cha.obs.source_service.html)
+for the [Open Build Service](http://openbuildservice.org/) which all
+assist with packaging source code from SCM (source code management)
+repositories into tarballs.  The authoritative source is
+https://github.com/openSUSE/obs-service-tar_scm.
+
+## Services
+
+### tar_scm *(deprecated)*
```

Comment:
hm, not sure if "deprecated" is maybe a bit too harsh .... what about just to move the section to the bottom and mark it as (not recommended) alternative?

Reply:
@adrianschroeter Are there still some use cases where `tar_scm` is the recommended choice over `obs_scm`?  If not, wouldn't there only be advantages to deprecating it, and no disadvantages?